config/msb/conf.d/fusionlog.yaml

```diff
@@ -4,7 +4,7 @@ data_dir : "/home/msb/msb_data"
 filename_datetime_fmt : "%Y%m%dT%H%M%S%z"
 logfile_interval : 3600  # temporal length in seconds of each logfile: 1h (3600s)
 key_order:
-    att : ["datetime", "epoch", "uptime", "roll", "roll_corr", "pitch", "pitch_corr"]
-    imu : ["datetime", "epoch", "uptime", "acc_x", "acc_y", "acc_z", "rot_x", "rot_y", "rot_z", "mag_x", "mag_y", "mag_z", "temp"]
-    gps : ["datetime", "epoch", "uptime", "gps_mode", "gps_timestamp", "latitude", "longitude", "altitude", "altitude_msl", "track", "mag_track", "mag_var", "speed"]
-    pwr : ["datetime", "epoch", "uptime", "load-1", "load-5", "load-15", "mem-available", "swap-cached"]
+    att : ["epoch", "uptime", "roll", "roll_corr", "pitch", "pitch_corr", "datetime"]
+    imu : ["epoch", "uptime", "acc_x", "acc_y", "acc_z", "rot_x", "rot_y", "rot_z", "mag_x", "mag_y", "mag_z", "temp", "datetime"]
+    gps : ["epoch", "uptime", "mode", "time", "latitude", "longitude", "altitude", "altitude_msl", "track", "mag_track", "mag_var", "speed", "datetime"]
+    sta : ["epoch", "uptime", "load-1", "load-5", "load-15", "mem-available", "swap-cached", "datetime"]
```

Comment:
what does sta stand for? status? if so this should not be changed is this branch

Reply:
done